cpp/bench/CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ if(BUILD_BENCH)
     bench/distance/distance_l1.cu
     bench/distance/distance_unexp_l2.cu
     bench/distance/fused_l2_nn.cu
+    bench/spatial/sparse_l2_nn.cu
```

Comment:
We've deprecated the `spatial` namespace in favor of `raft/neighbors`, though I also see the file below is in `bench/distance`. For example, we have the `fusedl2nn` primitive in `distance` so maybe the `masked-nn` belongs there as well. Once we have the `masked-knn` we can add that to neighbors (like the `fusedl2knn`).

Reply:
The masked_nn benchmark is now in distance and the CMakeLlists now has the correct location.